content/en/getting_started/synthetics/api_test.md

```diff
@@ -37,22 +40,25 @@ The example below demonstrates the creation of an [HTTP test][2], a subtype of [
     - Add the URL of the endpoint you want to monitor. If you don’t know what to start with, you can use `https://www.shopist.io/`, a test web application. Defining the endpoint to test automatically populates the name of your test to `Test on www.shopist.io`. You can change your test name to something else if you want to.
     - You can select **Advanced Options** to use custom request headers, authentication credentials, body content, or cookies.
     - You can set tags such as `env:prod` and `app:shopist` on your test. Tags allow you to keep your test suite organized and quickly find tests you're interested in on the homepage.
+    - You can create global variables and locally defined variables in your test options. If you need to use credentials or want to inject a dynamically defined timestamp in your API call, click **Create Local Variable**.
```

Comment:
```suggestion
    - You can use secure global variables, for instance if you need to use credentials in your API call. You can also create local variables, in case you want to inject a dynamically defined timestamp in your request payload. Once created, both variables can be injected in your test options by typing `{{` and selecting the variable of interest.
```

Let's also:
- Add a link to https://docs.datadoghq.com/synthetics/settings/?tab=specifyvalue#global-variables on global variables
- Add a link to https://docs.datadoghq.com/synthetics/api_tests/http_tests?tab=requestoptions#variables on local variables